### PR TITLE
Create

### PR DESCRIPTION
**Pull request by xh3b4sd**
Support a proper submit command.

### CODE REVIEW

**Comment by xh3b4sd** [comment]
RFR @JosephSalisbury @zeisss

File: cli/common.go [hunk 1]
```diff
@@ -74,13 +80,14 @@ func createSliceIDs(slices []string) ([]string, error) {
...
 
-func createRequest(slices []string) (controller.Request, error) {
+func createRequestWithContent(slices []string) (controller.Request, error) {
 	err := validateArgs(slices)
 	if err != nil {
 		return controller.Request{}, maskAny(err)
 	}
 
 	req := controller.Request{
+		Group:    dirnameFromSlices(slices),
```
Comment (zeisss) [inline cli/common.go:90]: thanks :cake:

File: cli/common_test.go [hunk 2]
```diff
@@ -14,10 +14,11 @@ type testFileSystemSetup struct {
 	FilePerm    os.FileMode
 }
 
-func Test_Common_createRequest(t *testing.T) {
+func Test_Common_createRequestWithContent(t *testing.T) {
 	testCases := []struct {
 		Setup    []testFileSystemSetup
 		Input    []string
+		Error    error
```
Comment (zeisss) [inline cli/common_test.go:24]: this is the expected error, right?

File: cli/common_test.go [hunk 3]
```diff
@@ -42,6 +44,15 @@ func Test_Common_createRequest(t *testing.T) {
 			},
 		},
 
+		// This test ensures that trying to load unit files with invalid input
```
Comment (zeisss) [inline cli/common_test.go:50]: I like the test - how about another one with a name as the Input but an empty file system?

**Comment by zeisss** [comment]
I like!

File: cli/common_test.go (hunk 2 above)
Comment (xh3b4sd) [inline cli/common_test.go:24]: Yes

File: cli/common.go (hunk 1 above)
Comment (xh3b4sd) [inline cli/common.go:90]: Yeah this is no way around. Funny you mentioned the need of it just a few hours ago.

File: cli/common_test.go (hunk 3 above)
Comment (xh3b4sd) [inline cli/common_test.go:50]: Will check. Shower and food first.

Comment (xh3b4sd) [inline cli/common_test.go:50]: Added

**Pull request merged**